goyaml.v3/emitterc.go

```diff
@@ -239,9 +239,23 @@ func yaml_emitter_increase_indent(emitter *yaml_emitter_t, flow, indentless bool
 		if emitter.states[len(emitter.states)-1] == yaml_EMIT_BLOCK_SEQUENCE_ITEM_STATE {
 			// The first indent inside a sequence will just skip the "- " indicator.
 			emitter.indent += 2
+			if compact_seq {
+				// The value compact_seq passed in is almost always set to `false` when this function is called,
+				// except when we are dealing with sequence nodes. So this gets triggered to subtract 2 only when we
+				// are increasing the indent to account for sequence nodes, which will be correct because we need to
+				// subtract 2 to account for the - at the beginning of the sequence node.
+				emitter.indent = emitter.indent - 2
+			}
```

Comment:
I still can't figure out how to exercise this path in a unit test

<img src="https://user-images.githubusercontent.com/980082/235240526-6cbc23ac-1aad-4f02-a65e-949f0b1dcdf7.png">

if we can't exercise this path, let's drop it for now, rather than possibly modify indentation in a way that will break something. We can always add it (with a test) if we figure out whether it's needed in this path as well.

Adjust dropping this in the `make sequence style configurable` commit

Reply:
Done.